lib/doorkeeper/config.rb

```diff
@@ -286,10 +291,15 @@ def option(name, options = {})
     option :base_controller,
            default: 'ActionController::Base'
 
-    attr_reader :reuse_access_token
     attr_reader :api_only
     attr_reader :enforce_content_type
 
+    ##
+    # Reuse of access tokens is only available when tokens are not hashed
+    def reuse_access_token
+      @reuse_access_token unless hash_secrets?
```

Comment:
I think it would be better to log some message like `"You are activated reuse access token with secrets encryptions that is unsupported. This option will be ignored"` instead of just silently return `false`

Reply:
I fully agree on that one